benchmarks/nemo/nemolite2d/manual_versions/psykal_cpp_ompss/Makefile

```diff
@@ -0,0 +1,77 @@
+# Makefile for manual C++ version of PSyKAl'd NEMOLite2D.
+#
+# This Makefile picks up the compiler to use any flags from
+# environment variables. e.g. to use gfortran:
+# 
+# export F90=gfortran
+# export CXX=g++
+# export CFLAGS="-O3"
+# export F90FLAGS="-O3"
+# export OMPFLAGS=" -fopenmp"
+# export AR=ar
+
+.PHONY: all nemolite2d_ompss
+.DEFAULT_GOAL := nemolite2d_ompss
+
+include ../common.mk
+
+# INF_LIB is an archive that must come at the end of the list of objects
+# passed to the linker
+LIBS = ${COMMON_LIB} ${TIMER_LIB} ${INF_LIB}
+COMMON_MODULES = time_step_mod.o ${COMMON_LIB} ${KERNELS} ${INF_LIB}
+
+# Normal targets
+all: nemolite2d_ompss
+
+nemolite2d_ompss: ${COMMON_LIB} timer_lib_parallel inf_lib time_step_ompss.o
+	${MAKE} MODULE_LIST="nemolite2d.o ${COMMON_MODULES} time_step_ompss.o" nemolite2d_ompss.exe
+
+nemolite2d.o: $(COMMON_MODULES)
+
+# Interdependencies between modules, alphabetical order
+time_step_mod.o: ${COMMON_LIB} inf_lib momentum_mod.o continuity_mod.o \
+                 time_update_mod.o boundary_conditions_mod.o
+
+# Linking the executable needs the -lstdc++ to link also the std lib functions
```

Comment:
Presumbaly we can remove this commented-out section?

Reply:
Done